apps/etterlatte-behandling/src/main/kotlin/behandling/tilbakekreving/TilbakekrevingServiceImpl.kt

```diff
@@ -0,0 +1,51 @@
+package no.nav.etterlatte.behandling.tilbakekreving
+
+import no.nav.etterlatte.kafka.JsonMessage
+import no.nav.etterlatte.kafka.KafkaProdusent
+import no.nav.etterlatte.libs.common.logging.getCorrelationId
+import no.nav.etterlatte.libs.common.rapidsandrivers.CORRELATION_ID_KEY
+import no.nav.etterlatte.libs.common.rapidsandrivers.TEKNISK_TID_KEY
+import no.nav.etterlatte.libs.common.tilbakekreving.StatistikkTilbakekrevingDto
+import no.nav.etterlatte.libs.common.tilbakekreving.TILBAKEKREVING_STATISTIKK_RIVER_KEY
+import no.nav.etterlatte.libs.common.tilbakekreving.TilbakekrevingHendelseType
+import no.nav.etterlatte.libs.common.tilbakekreving.lagEventnameForType
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+import java.time.LocalDateTime
+
+interface ITilbakekrevingService {
+    fun sendTilbakekreving(
+        statistikkTilbakekreving: StatistikkTilbakekrevingDto,
+        type: TilbakekrevingHendelseType,
+    )
+}
+
+class TilbakekrevingServiceImpl(
+    private val rapid: KafkaProdusent<String, String>,
+) : ITilbakekrevingService {
+    private val logger: Logger = LoggerFactory.getLogger(this::class.java)
+
+    override fun sendTilbakekreving(
+        statistikkTilbakekreving: StatistikkTilbakekrevingDto,
+        type: TilbakekrevingHendelseType,
```

Comment:
nitpick men type ligger vel på dto?

Reply:
Ikke med de andre endringene over, for da sender jeg bare TilbakekrevingBehandling over linja